felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -1182,8 +1190,10 @@ func (m *bpfEndpointManager) onInterfaceUpdate(update *ifaceStateUpdate) {
 			// We require host interfaces to be in non-strict RPF mode so that
 			// packets can return straight to host for services bypassing CTLB.
 			switch update.Name {
-			case dataplanedefs.BPFInDev, dataplanedefs.BPFOutDev:
-				// do nothing
+			case dataplanedefs.BPFOutDev:
```

Comment:
what happes to "do nothing" with dataplanedefs.BPFInDev ? Aren't you already handling this case above?